test/fixtures/invalid-es5.js

```diff
@@ -92,7 +92,7 @@ var APP;
 		}
 
 		// eslint-disable-next-line no-trailing-spaces
-		options.spaces = 'foo'; 
```

Comment:
This is meant to have a trailing space, right?

Reply:
Whoops, yes, fixed.